src/neptune/internal/init/project.py

```diff
@@ -34,8 +34,8 @@
 
 
 def init_project(
-    *,
     project: Optional[str] = None,
+    *,
```

Comment:
Did you applied it here: https://github.com/neptune-ai/neptune-client/pull/1228 as well?

Reply:
No, I've messed with branches.
But let's add it here.

Reply:
Ok